### PR TITLE
Cleanups + few fixes

### PR DESCRIPTION
**Pull request by igor-dv**
## What I did
- [x] remove `babel-loader@^8.0.0-beta.6`
- [x] upgrade `@babel/*` deps
- [x] ignore `CHANGELOG.md` from a linter
- [x] fix `angular-cli` netlify build
- [x] (vue peer deps range) https://github.com/storybooks/storybook/issues/4186

### CODE REVIEW

File: lib/cli/test/fixtures/meteor/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "start": "meteor run"
   },
   "dependencies": {
-    "@babel/runtime": "7.0.0-rc.2",
+    "@babel/runtime": "7.0.0",
```
Comment (pksunkara) [inline lib/cli/test/fixtures/meteor/package.json]: Why no caret here?

File: lib/cli/test/fixtures/react/package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     "react-dom": "^15.6.1"
   },
   "devDependencies": {
-    "@babel/plugin-external-helpers": "7.0.0-rc.2",
-    "@babel/preset-react": "7.0.0-rc.2",
+    "@babel/plugin-external-helpers": "7.0.0",
+    "@babel/preset-react": "7.0.0",
```
Comment (pksunkara) [inline lib/cli/test/fixtures/react/package.json]: Again, carets.

**Review by pksunkara — CHANGES_REQUESTED** [review]
Can we use `^` more please?

**Comment by igor-dv** [comment]
Yeah, I don't know why it was like this 🤷‍♂️  I can change it

**Review by pksunkara — APPROVED** [review]

**Review by ndelangen — APPROVED** [review]

**Pull request merged**